src/main/java/org/reactome/release/qa/common/QACheckerHelper.java

```diff
@@ -19,6 +19,13 @@
     
     public static final String IS_NOT_NULL = "IS NOT NULL";
     public static final String IS_NULL = "IS NULL";
+
+    // Variables for CoV-specific QA tests, from the CoV-1-to-CoV-2 projection process (August 2020).
+    private static final long cov2InfectionPathwayDbId = 9694516L;
+    private static final long cov1SpeciesDbId = 9678119L;
+    private static final long cov2SpeciesDbId = 9681683L;
+    private static final long cov1DiseaseDbId = 9678120L;
+    private static final long cov2DiseaseDbId = 9683912L;
```

Comment:
Since these are class-level constants, could their names be uppercase with underscores?

Reply:
Done